src/EventStore.js

```diff
@@ -9,7 +9,13 @@ class EventStore extends Store {
   constructor (ipfs, id, dbname, options = {}) {
     if (options.Index === undefined) Object.assign(options, { Index: EventIndex })
     super(ipfs, id, dbname, options)
-    this._type = 'eventlog'
+    this._type = 'eventlog';
+    this.events.on("replicated.progress", (address, hash, entry, progress, have) => {
```

Comment:
Now that the `orbit-db-store` PR has merged, will we get `procEntry` for free here? i.e. do we not need to duplicate this code here?

Reply:
Exactly. We don't need to duplicate code.

Reply:
Will approve after this :+1: